website/docs/docs/build/metricflow-cli.md

```diff
@@ -367,3 +367,11 @@ mf query --metrics order_amount --group-by metric_time,is_food_order --limit 10
 
 </TabItem>
 </Tabs>
+
+## Time Granularity
+You can optionally specify the time granularity you want your data to be aggregated at by appending double underscore and the unit of granularity you want to the global time dimension `metric_time`. The granularity options to group by are: day, week, month, quarter year. 
```

Comment:
```suggestion
Optionally, you can specify the time granularity you want your data to be aggregated at by appending two underscores and the unit of granularity you want to `metric_time`, the global time dimension . You can group the granularity by: `day`, `week`, `month`, `quarter`, and `year`. 
```

Reply:
i added ticks around the groups because i understood it to mean the "text" to use after the "__". please ignore if i'm mistaken